Documentation/contributing/development/contributing_guide.rst

```diff
@@ -183,13 +213,63 @@ Getting a pull request merged
 
 #. Owners of the repository will automatically adjust the labels on the pull
    request to track its state and progress towards merging.
+
 #. Once the PR has been reviewed and the CI tests have passed, the PR will be
    merged by one of the repository owners. In case this does not happen, ping
    us on Slack.
 
-
-Pull request review process
----------------------------
+#. If reviewers have requested for changes and once those changes are done,
```

Comment:
Some contributors seem unsure how to push changes to answer to a review (because `git push --force` is often depicted as a bad thing, some people hesitate to force-push to the PR). Should we add a note explaining that force-pushing to the PR branch is the usual workflow, or something like that?

Reply:
I think that sounds reasonable. If we do, I think we should encourage the use of `--force-with-lease` instead as an opportunity to share its existence. :)